### PR TITLE
Update to v11, add collapse to R memory benchmark

### PR DESCRIPTION
**Pull request by arthurgailes**
I got curious on how collapse would perform, and it turns out quite well. Also updated polars syntax, which was broken in the v11 update in the book.

### CODE REVIEW

**Comment by eitsupi** [comment]
@ddotta Any update on this?

**Comment by ddotta** [comment]
@eitsupi and @arthurgailes Sorry I don't have enough time at the moment to update the cookbook. I'll do my best but I'm looking for contributors and PRs 😃

**Comment by eitsupi** [comment]
> I'm looking for contributors and PRs

I am not sure what you are trying to say here.
Is this not PR?

File: book/content/benchmarking/_from_r_object.qmd [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 This section analyses the different methods for making a query **from an R object already loaded in memory**.
 
-Let's start by comparing polars with R base, dplyr and data.table.
+Let's start by comparing polars with R base, dplyr and data.table. We'll alsso add collapse, a recent package that is very fast for data manipulation.
```
Comment (ddotta) [inline book/content/benchmarking/_from_r_object.qmd:5]:
```suggestion
Let's start by comparing polars with R base, dplyr and data.table. We'll also add collapse, a recent package that is very fast for data manipulation.
```

**Review by ddotta — APPROVED** [review]

**Comment by arthurgailes** [comment]
Thank you; your work here is excellent.

**Pull request merged**